app/javascript/mastodon/features/account_timeline/components/moved_note.js

```diff
@@ -23,7 +23,7 @@ export default class MovedNote extends ImmutablePureComponent {
         </div>
 
         <div className='moved-account-banner__action'>
-          <Link to={`/@${to.get('acct')}`} className='detailed-status__display-name'>
+          <Link rel='me' href={to.get('url')} to={`/@${to.get('acct')}`} className='detailed-status__display-name'>
```

Comment:
The `to=` property already renders a `href` and I doubt multiple href are allowed. Also this entire component is not rendered serverside, so the resulting HTML contains neither the link nor your fixed version of it. I don't think this will work.

Reply:
you likely want to patch this file instead: https://github.com/mastodon/mastodon/blob/main/app/views/accounts/show.html.haml#L8-L9